x-pack/plugins/monitoring/public/application/pages/loading_page.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import { i18n } from '@kbn/i18n';
+import { PageTemplate } from './page_template';
+import { PageLoading } from '../../components';
+import { useClusters } from '../hooks/use_clusters';
+import { CODE_PATH_ELASTICSEARCH } from '../../../common/constants';
+
+const CODE_PATHS = [CODE_PATH_ELASTICSEARCH];
+
+export const LoadingPage = () => {
```

Comment:
This is the migration for the [loading route](https://github.com/elastic/kibana/blob/master/x-pack/plugins/monitoring/public/views/loading/index.js)